manage.py

```diff
@@ -289,11 +289,11 @@ def config(config_key):
 
     if config_key:
         # Remap None values to an empty string
-        print(app.config.get(config_key, '') or '')
+        print((app.config.get(config_key, '') or ''))
```

Comment:
isn't that a redundant `or ''`, as you provide that as a default to `get()`?

Reply:
the default to `get()` will work if it's not in `app.config`, but there are some config values that are set to `None`. The `or ''` is to catch config values set as `None` and return `''` instead.